frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { match as RouterMatch } from 'react-router';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { Alert, Button } from '@patternfly/react-core';
+import { StorageClassResourceKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { history } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { fetchK8s } from '@console/internal/graphql/client';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager';
+import { LSO_NAMESPACE } from '@console/local-storage-operator-plugin/src/constants';
+import { CreateOCS } from './install-lso-sc';
+import { scResource, LSOResource } from '../../../constants/resources';
+import { filterNoProvSC } from '../../../utils/install';
+
+import './attached-devices.scss';
+
+export const CreateOCSOnAttachedDevices: React.FC<CreateOCSOnAttachedDevicesProps> = ({
+  match,
+}) => {
+  const [isNoProvSCPresent, SetIsNoProvSCPresent] = React.useState(false);
+  // LSO stands for local-storage-operator
+  const [LSOEnabled, setLSOEnabled] = React.useState(false);
+  const [scData, scLoaded, scLoadError] = useK8sWatchResource<StorageClassResourceKind[]>(
+    scResource,
+  );
+  const [LSOData, LSOLoaded, LSOLoadError] = useK8sWatchResource<K8sResourceKind>(LSOResource);
```

Comment:
```suggestion
  const [lsoSubscription, lsoSubscriptionLoaded, lsoSubscriptionLoadError] = useK8sWatchResource<K8sResourceKind>(LSOResource);
```

Reply:
LSO is preferable here instead `lso` like we have OCS and not `ocs`

Reply:
I agree there are places where I have used `lvs`, needs to fix that in follow up.

Reply:
Its a subscription, thats what I pointed.

Reply:
As we OCS and not ocs, similary we should have LSO and not lso.